server/db/mockData/seedData.js

```diff
@@ -125,6 +125,30 @@ let seedData = {
       shoutee: 2,
       company_value: 1,
     },
+    {
```

Comment:
Is this seeded data supposed to be duplicated like this?

Reply:
They have different dates, but I'll update the other properties so they aren't so similar.

Reply:
It may not hurt to have a larger set to work with either, like 30 or 40 shoutouts perhaps? It would allow us at least a few per month for the last 6 months for example, and better show off how the tables and graphs work